p2pd/main.go

```diff
@@ -37,6 +38,10 @@ func main() {
 	pubsubSignStrict := flag.Bool("pubsubSignStrict", false, "Enables pubsub strict signature verification")
 	gossipsubHeartbeatInterval := flag.Duration("gossipsubHeartbeatInterval", 0, "Specifies the gossipsub heartbeat interval")
 	gossipsubHeartbeatInitialDelay := flag.Duration("gossipsubHeartbeatInitialDelay", 0, "Specifies the gossipsub initial heartbeat delay")
+	relayEnabled := flag.Bool("relay", false, "Enables a relay")
+	relayActive := flag.Bool("relayAct", false, "Enables active mode on the relay")
```

Comment:
call this option `relayActive1, no need to cut down words like this.

Reply:
Switched all of the relay* names to be full words